### PR TITLE
Fix strict template errors in modeling assessment

### PR DESCRIPTION
**Pull request by iozbeyli**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### CODE REVIEW

**Review by SimonEntholzer — APPROVED** [review]
Changes lgtm and changed components work fine on TS1

**Review by ivanchimeno — APPROVED** [review]
Code looks good

File: src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class ModelingAssessmentComponent implements AfterViewInit, OnDestroy, On
     @Input() highlightedElements: Map<string, string>; // map elementId -> highlight color
     @Input() centeredElementId: string;
     @Input() feedbacks: Feedback[] = [];
-    @Input() diagramType: UMLDiagramType;
+    @Input() diagramType?: UMLDiagramType = UMLDiagramType.ClassDiagram;
```
Comment (krusche) [inline src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts:26]: if you have a default value, why do you still need `?`

**Review by ivanchimeno — APPROVED** [review]
re-approving

**Review by SimonEntholzer — APPROVED** [review]
Looks good

**Pull request merged**